client/landing/stepper/declarative-flow/internals/steps-repository/components/flow-card/index.tsx

```diff
@@ -27,7 +33,10 @@ const FlowCard = ( { icon, onClick, text, title, disabled = false }: FlowCardPro
 						</FlexItem>
 					) }
 					<FlexBlock>
-						<h3 className="flow-question__heading">{ title }</h3>
+						<div className="flow-question__title-wrapper">
+							<h3 className="flow-question__heading">{ title }</h3>
+							{ badge && <Badge type={ badge.type }>{ badge.text }</Badge> }
+						</div>
```

Comment:
```suggestion
						<h3 className="flow-question__heading">
							{ title }
							{ badge && <Badge type={ badge.type }>{ badge.text }</Badge> }
						</h3>
```

I think we should maintain the structure as much as possible to avoid incompatibilities.